packages/components/src/SegmentedControl/SegmentedControl.module.css

```diff
@@ -0,0 +1,62 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(var(--segmentedControl--option-count), 1fr);
+  align-items: center;
+  position: relative;
+  min-width: max-content;
+  border-radius: var(--radius-base);
+  border: var(--border-base) solid var(--color-border--interactive);
+  background-color: var(--color-surface--background);
+}
+
+.container input[type="radio"] {
+  position: absolute;
+  left: -999vw;
+  cursor: pointer;
+}
+
+.container label {
+  z-index: 1;
+  padding: var(--space-smaller) 12px;
+  overflow: hidden;
+  font-size: var(--typography--fontSize-base);
+  font-weight: 600;
+  text-align: center;
+  cursor: pointer;
+  align-items: center;
+}
+
+.container span {
+  position: absolute;
+  left: 0;
+  width: calc(100% / var(--segmentedControl--option-count));
+  height: 100%;
+  box-shadow: var(--shadow-low);
+  border-radius: var(--radius-base);
+  background-color: var(--color-surface);
+  transition: var(--timing-base) left ease;
+}
+
+.container input[type="radio"]:focus-visible ~ span {
+  box-shadow: var(--shadow-focus);
+}
+
+.container input[type="radio"]:nth-of-type(1):checked ~ span {
+  left: 0;
```

Comment:
This is a huge nitpick but setting this to 0 leaves us with a "double border" at the left edge of the component, so can we use

```suggestion
  left: -1px;
```

here? _(OR if there's some other modification we should make to box-sizing, etc that would get us the same result, ultimately we just want to avoid the double-border)_

**Current**
![image](https://github.com/user-attachments/assets/631e2c2e-ca4d-4fb0-a7d5-24710f732d72)

**What we want**
![image](https://github.com/user-attachments/assets/7db3c409-fe38-4ea2-b65e-5fd686f9a123)

Reply:
I love a _nit by chris_ ™️ 
Used `box-sizing` to fix this - setting `-1px` actually cut off some of the left border

<img width="333" alt="Screenshot 2024-10-23 at 5 20 01 PM" src="https://github.com/user-attachments/assets/343b1f54-875e-4214-9f44-73507056d0a4">

Reply:
Oh nice 👌